HarpGuru/helpers/setGlobalReactNState/testSetup.tsx

```diff
@@ -1,13 +1,24 @@
+import type { GlobalTuple } from 'reactn/types/use-global'
+import type { State } from 'reactn/default'
 import { useGlobal } from 'reactn'
 import React from 'react'
 import { render } from '@testing-library/react-native'
 
-export const setup = (): Record<'0', Record<string, unknown>> => {
-  const returnVal = { '0': {} }
+export const setup = (): Record<'globalTuple', GlobalTuple<State>> => {
+  const blankState = {} as State
+  const globalWrapper = {
+    globalTuple: [
+      blankState,
+      (arg0: State) => {
+        console.log(arg0)
+      },
+    ],
+  } as Record<'globalTuple', GlobalTuple<State>>
   const TestComponent = (): null => {
-    Object.assign(returnVal, useGlobal())
+    const globalTuple: GlobalTuple<State> = useGlobal()
+    globalWrapper.globalTuple = globalTuple
     return null
   }
   render(<TestComponent />)
-  return returnVal
+  return globalWrapper
 }
```

Comment:
This file needs to be renamed since it's used to check the global state from a non-component context.

Something like StateInformantComponent seems on the nose.

Reply:
Done